helper/iplookuptable.c

```diff
@@ -678,8 +679,9 @@ odph_iplookup_table_put_value(odph_table_t tbl, void *key, void *value)
 
 	nexthop = *((odp_buffer_t *)value);
 
-	if (prefix->cidr == 0)
+	if (prefix->cidr == 0 || prefix->cidr == 255)
 		return -1;
+
 	prefix->ip = prefix->ip & (0xffffffff << (IP_LENGTH - prefix->cidr));
```

Comment:
Given this statement, isn't the valid range for `prefix->cidr` 0..31? In that case the above check for `prefix->cidr == 255` seems off.

Reply:
looks like yes, and it's also will be good to add to other function put,get,remove validation of passed values.